salesagility/SuiteCRM
# Feature/acceptance testing

## Description



- Adds basic acceptance testing to every default module
- Fixes PHP errors preventing certain modules from being visible

## How To Test This

See travis or run locally:

./vendor/bin/codecept run acceptance --env chrome-driver

## Types of changes


### Final checklist
